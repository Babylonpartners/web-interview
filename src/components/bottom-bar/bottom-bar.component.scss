.bottom-bar {
  margin: 30rem 0 12rem;

  .button {
    width: 100%;
  }
}
