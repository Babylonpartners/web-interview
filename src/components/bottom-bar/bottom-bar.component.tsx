import React, { FunctionComponent } from 'react'

import './bottom-bar.component.scss'

export const BottomBar: FunctionComponent = props => {
  return <div className="bottom-bar">{props.children}</div>
}
