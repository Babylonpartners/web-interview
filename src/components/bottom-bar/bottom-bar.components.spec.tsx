import React from 'react'
import renderer from 'react-test-renderer'

import { BottomBar } from './bottom-bar.component'

describe('BottomBar', () => {
  it('should match snapshot', () => {
    const component = renderer.create(<BottomBar />)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
