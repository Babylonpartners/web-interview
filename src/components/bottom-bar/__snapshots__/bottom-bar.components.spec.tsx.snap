// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBar should match snapshot 1`] = `
<div
  className="bottom-bar"
/>
`;
